frontend/src/app/components/main-menu/main-menu-toggle.service.ts

```diff
@@ -117,7 +117,7 @@ export class MainMenuToggleService {
   public closeMenu():void {
     this.setWidth(0);
     window.OpenProject.guardedLocalStorage(this.localStorageStateKey, 'true');
-    jQuery('.wp-query-menu--search-input').blur();
+    jQuery('.collapsible-menu--search-input').blur();
```

Comment:
Chapeau for renaming the css selector. Way more applicable that way. :+1:

Reply:
Thanks 👯